<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATLAS Simulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MplWidget" name="mpl" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>791</width>
      <height>411</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitBtn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>620</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Step</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoRepeatInterval">
     <number>500</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="stepSize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>56</width>
      <height>23</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="startStop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="stepsizelabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>580</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>step</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>30</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>470</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Submitted Jobs:</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>411</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>t=0</string>
    </property>
   </widget>
   <widget class="QSlider" name="simspeedSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>160</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>50</number>
    </property>
    <property name="value">
     <number>350</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="simspeedLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>250ms delay</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>460</y>
      <width>381</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="statusLayout"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
